src/amo/components/Header/index.js

```diff
@@ -60,12 +84,31 @@ export class HeaderBase extends React.Component {
           <DownloadFirefoxButton
             className="Header-download-button Header-button"
           />
-          <AuthenticateButton
-            className="Header-authenticate-button Header-button Button--action
+
+          {isAuthenticated ? (
+            <DropdownMenu
+              title={username}
+              className="Header-authenticate-button Header-button"
+            >
+              <DropdownMenuItem title={i18n.gettext('Actions')} />
+              <DropdownMenuItem
+                title={i18n.gettext('Add-ons Home Page')}
```

Comment:
This is maybe not the best link, but the "log out" alone does not look nice 😞

Reply:
Ah, I see what you mean now. So in the future I think stuff like "My Profile" or "My collections" would work. Linking to "View Profile" and/or "Edit Profile/Settings" for now I think is better than "Add-ons Home Page". Sure, it will fallback to the old addons-server code but I think that's a better link.